docs/src/pages/components/slider/slider.md

```diff
@@ -14,6 +14,12 @@ Sliders reflect a range of values along a bar, from which users may select a sin
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
+## Sizes
```

Comment:
I wonder about this header. Half of the time we use `Sizes`, the other half `Size`.

Reply:
`Sizes` makes more sense to me. As a user, I want to see all of the sizes in the demo to see what is the different between them.